packages/carbon-graphs/src/js/controls/PairedResult/PairedResult.js

```diff
@@ -78,6 +78,27 @@ const loadInput = (inputJSON) => new PairedResultConfig()
   .clone()
   .getConfig();
 
+/**
+ * Filters invalid data like null and blank.
+ *
+ * @private
+ * @param {object} data - Data points object
+ * @returns {object} filteredData - filtered data object
+ */
+const filterPairedResultData = (data) => {
+  let filteredData = [];
+  data.map((value) => {
+    let temp = {};
+    iterateOnPairType((t) => {
+
+      if(value[t] != null && !(typeof value[t] === 'object' && Object.keys(value[t]).length === 0)){
```

Comment:
Is there any utility function present in carbon-graphs to check the value is not null and not object?

Reply:
Can [notEmpty](https://github.com/cerner/carbon-graphs/blob/main/src/main/js/helpers/utils.js#L87) work here?

Reply:
**notempty** will check things like undefined, null, blank when it is a string and blank when it is an object. For this instance in here we only need to check two things like null and blank when object so that is the reason why did not use notempty in here.